app/models/world_location.rb

```diff
@@ -32,6 +32,8 @@ class WorldLocation < ApplicationRecord
 
   scope :ordered_by_name, -> { with_translations(I18n.default_locale).order("world_location_translations.name") }
 
+  after_commit :republish_world_index_page_to_publishing_api
```

Comment:
Now that we have this, do we also need to update the seeds file to skip this (e.g. as per [this](https://github.com/alphagov/whitehall/blob/c3a35269ce989b01c1c8e94da13394b5e1b03591/db/seeds.rb#L90)) - this was noticed by new devs setting up whitehall e.g. [here](https://gds.slack.com/archives/C02L13S214K/p1679061305102379?thread_ts=1679057170.131029&cid=C02L13S214K) from some of our recent work, and I think this might cause the same issue as we have world locations in the seeds?